dolphin/processor/config.py

```diff
@@ -490,15 +482,24 @@ def get_index_lens_light_model_list(self):
          (for multiple filters)
         """
         if 'lens_light' in self.settings['model']:
-            index_lens_light_model_list = []
-            index_num = 0
-            for i in range(self.band_number):
-                single_index_list = []
-                for j in range(len(self.settings['model']['lens_light'])):
-                    single_index_list.append(index_num)
-                    index_num += 1
-                index_lens_light_model_list.append(single_index_list)
-            return index_lens_light_model_list
+            if self.band_number == 1:
+                index_list = [[]]
+                for k, model in enumerate(
+                        self.settings['model']['lens_light']):
+                    index_list[0].append(k)
+                return index_list
+            else:
+                if 'lens_light_band_index' in self.settings['model']:
+                    index_list_settings = \
```

Comment:
It is unnecessary to create a new variable here. You can simply use `self.settings['model']['lens_light_band_index']` for that one time you need it in line 496.